projects/core/components/dialog/dialog.component.ts

```diff
@@ -108,3 +99,7 @@ export class TuiDialogComponent<O, I> {
         }
     }
 }
+
+function toObservable<T>(valueOrStream: Observable<T> | T): Observable<T> {
```

Comment:
maybe move to `@taiga/cdk/utils`

Reply:
\+ reuse this new util `tuiToObservable` for the similar cases:
<img width="821" alt="Screenshot 2023-03-14 at 10 23 48" src="https://user-images.githubusercontent.com/35179038/224925811-a6c3a200-000e-4ab9-a64a-aff8d6edd8ab.png">

Reply:
i18n does not depend on cdk though

Reply:
> i18n does not depend on cdk though

my mistake)

Reply:
It is only used here, I'll keep it here for now.